maintainer/walberla_kernels/templates/FieldAccessors.tmpl.h

```diff
@@ -600,8 +604,14 @@ namespace PressureTensor
          Cell const & cell )
    {
         const {{dtype}} & xyz0 = pdf_field->get(cell, uint_t{ 0u });
+        const int x = cell[0];
+        const int y = cell[1];
+        const int z = cell[2];
         {% for i in range(Q) -%}
-            const {{dtype}} f_{{i}} = pdf_field->getF( &xyz0, uint_t{ {{i}}u });
+            const {{dtype}} f_{{i}} = pdf_field->get({{inv_neighbor_dir[i][0]}} + x,
+                                                     {{inv_neighbor_dir[i][1]}} + y,
```

Comment:
Why do we have to acces the neighbors rather than the actual cell here and in the other accessors? Hasn't the streaming alredy taken place in the StreamCollide sweep?